assets/js/admin.js

```diff
@@ -198,15 +198,20 @@ function populateTemplate( item, dialog, action ) {
 					</form>
 					<promote-job-template>
 						<div slot="buttons" class="promote-buttons-group">
-								<a class="promote-button button button-primary" target="_blank" href="${ this.getAttribute( 'data-href' ) }">${ job_manager_admin_params.job_listing_promote_strings.promote_job }</a>
-								<a class="promote-button button button-secondary" target="_blank" href="#">${ job_manager_admin_params.job_listing_promote_strings.learn_more }</a>
+							<a id="wpjm-promote-button" class="promote-button button button-primary" target="_blank" rel="noopener noreferrer" href="${ this.getAttribute( 'data-href' ) }">${ job_manager_admin_params.job_listing_promote_strings.promote_job }</a>
+							<a class="promote-button button button-secondary" target="_blank" rel="noopener noreferrer" href="#">${ job_manager_admin_params.job_listing_promote_strings.learn_more }</a>
```

Comment:
Won't the Learn More button have the same requirement to close the modal? :thinking:

Reply:
Good question! I don't think so. My thought is that the user will try to promote a job, but before that, they just want to understand how it works, so they go to a different tab, and return continuing from where they stopped.

This is different for the promote button because it changes the state of the job. It wouldn't make sense to click again on "Promote". Ideally, it should also update the job status on the listing, but I think only closing the modal is fine for now.

Does it make sense to you? Or do you have any other thoughts on that?